File: app/src/main/java/com/dertyp7214/appstore/LocalJSON.java
/*
 * Copyright (c) 2018.
 * Created by Josua Lengwenath
 */

package com.dertyp7214.appstore;

import android.content.Context;
import android.content.SharedPreferences;

public class LocalJSON {

    public static String getJSON(Context context){
        return context.getSharedPreferences("json", Context.MODE_PRIVATE).getString("json", "{\"error\": true}");
    }

    public static void setJSON(Context context, String json){
        context.getSharedPreferences("json", Context.MODE_PRIVATE).edit().putString("json", json).apply();
    }

}

File: app/src/main/java/com/dertyp7214/appstore/LocalJSON.kt
/*
 * Copyright (c) 2018.
 * Created by Josua Lengwenath
 */

@file:Suppress("NULLABILITY_MISMATCH_BASED_ON_JAVA_ANNOTATIONS")

package com.dertyp7214.appstore

import android.content.Context
import androidx.core.content.edit

object LocalJSON {

    fun getJSON(context: Context): String {
        return context.getSharedPreferences("json", Context.MODE_PRIVATE).getString("json", "{\"error\": true}")
    }

    fun setJSON(context: Context, json: String) {
        context.getSharedPreferences("json", Context.MODE_PRIVATE).edit {
            json to "json"
        }
    }
}
